shared/actions/chat.js

```diff
@@ -1133,29 +1133,32 @@ function _unboxedToMessage (message: MessageUnboxed, idx: number, yourName, your
           }
         }
         case CommonMessageType.delete:
+          const deletedIDs = payload.messageBody.delete && payload.messageBody.delete.messageIDs || []
```

Comment:
Flow complains unless this is hoisted to a `const`:

```
actions/chat.js:1141
1141:             deletedIDs: payload.messageBody.delete && payload.messageBody.delete.messageIDs || [],
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^ property `delete` of unknown type. This type is incompatible with
107:   deletedIDs: Array<MessageID>,
                   ^^^^^^^^^^^^^^^^ array type. See: constants/chat.js:107

actions/chat.js:1141
1141:             deletedIDs: payload.messageBody.delete && payload.messageBody.delete.messageIDs || [],
                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ property `messageIDs` of unknown type. This type is incompatible with
107:   deletedIDs: Array<MessageID>,
                   ^^^^^^^^^^^^^^^^ array type. See: constants/chat.js:107


Found 2 errors
```

These changes seem fine to me, but this flow behavior is mystifying to me. Any ideas @MarcoPolo?